src/WCSimPMTObject.cc

```diff
@@ -220,7 +219,27 @@ G4float PMT20inch::GetmaxQE(){
   return maxQE;
 }
 
+// Should be actual PMT Dark Rate, not effective dark rate in detector including other LE noise
+G4float PMT20inch::GetDarkRate(){
+  /* From e-mail discussion with A.Konaka and S.Nakayama:
+   * SK-I: 4.2 kHz 
+   * SK-IV:5.7 kHz, both before electronics threshold in skdetsim
+   * Measured DN with 0.25 pe threshold:
+   * SK-I: 3.4 kHz  (2003 SK-NIM: 3 kHz. A.Konaka: "2kHz with hot PMTs removed?") 
+   * SK-IV: 4.5 kHz (higher due to FRP)
+   * ToDo: investigate after updating electronics routing, whether to change value to 3.4 kHz
+   */
+
```

Comment:
That makes sense. I'll move the ToDo in that comment to a new issue.
The rest of the comment serves as reference/documentation of where the numbers
come from, which is the discussion we had over e-mail. Unless there is a better place
to document measurements/numbers used in the code.